src/css-grids/cssgrids.html

```diff
@@ -0,0 +1,97 @@
+        </div>
+        <div class="Calculator" ></div> 
+          <table border="1"> 
+             <tr> 
+                <td colspan="3"><input type="text" id="result"/></td> 
+                <td><input type="button" value="c" onclick="clr()"/> </td> 
+             </tr> 
+             <tr> 
+                <td><input type="button" value="1" onclick="dis('1')"/> </td> 
+                <td><input type="button" value="2" onclick="dis('2')"/> </td> 
+                <td><input type="button" value="3" onclick="dis('3')"/> </td> 
+                <td><input type="button" value="/" onclick="dis('/')"/> </td> 
+             </tr> 
+             <tr> 
+                <td><input type="button" value="4" onclick="dis('4')"/> </td> 
+                <td><input type="button" value="5" onclick="dis('5')"/> </td> 
+                <td><input type="button" value="6" onclick="dis('6')"/> </td> 
+                <td><input type="button" value="-" onclick="dis('-')"/> </td> 
+             </tr> 
+             <tr> 
+                <td><input type="button" value="7" onclick="dis('7')"/> </td> 
+                <td><input type="button" value="8" onclick="dis('8')"/> </td> 
+                <td><input type="button" value="9" onclick="dis('9')"/> </td> 
+                <td><input type="button" value="+" onclick="dis('+')"/> </td> 
+             </tr> 
+             <tr> 
+                <td><input type="button" value="." onclick="dis('.')"/> </td> 
+                <td><input type="button" value="0" onclick="dis('0')"/> </td> 
+                <td><input type="button" value="=" onclick="solve()"/> </td> 
+                <td><input type="button" value="*" onclick="dis('*')"/> </td> 
+             </tr> 
+          </table> 
+        <div class="box-b"></div>
+            <div class="date">
+                <h1>1/30/2020</h1>
+            </div>
+            <div class="analog-clock">
+                <svg width="140" height="140">
+                    <circle id="clock-face" cx="70" cy="70" r="65" />
+                    <line id="h-hand" x1="70" y1="70" x2="70" y2="38" />
+                    <line id="m-hand" x1="70" y1="70" x2="70" y2="20" />
+                    <line id="s-hand" x1="70" y1="70" x2="70" y2="12" />
+                    <line id="s-tail" x1="70" y1="70" x2="70" y2="56" />
+                    <text x="62" y="18">12</text>
+                    <text x="126" y="76">3</text>
+                    <text x="66" y="130">6</text>
+                    <text x="7" y="76">9</text>
+                </svg>
```

Comment:
dangggg. above and beyond. keep it simple next time! looks good though and hope you learned a little bit about svg's.